pkg/kv/kvserver/mvcc_gc_queue.go

```diff
@@ -288,9 +288,10 @@ func makeMVCCGCQueueScore(
 	gcTTL time.Duration,
 	canAdvanceGCThreshold bool,
 ) mvccGCQueueScore {
-	repl.mu.Lock()
```

Comment:
Is it safe to downgrade this to a read lock? I wonder whether it was `Lock` just by accident / because it's an old code.

Reply:
Don't see why not.